### DISCUSSION TITLE
Prediktif analisis menggunakan metode LSTM di R Studio

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#292] [Q&A] [answered]
Bagaimana langkah-langkah untuk melakukan analisis prediktif menggunakan metode LSTM untuk data time series di R studio ?

**Comment by NabiilahArdini** [comment, marked as answer]
Dear @yohanespm77 , 
berikut salah satu contoh artikel yang menjelaskan cara melakukan LSTM untuk Time Series di R. 
https://algotech.netlify.app/blog/time-series-prediction-with-lstm/
Semoga dapat bermanfaat!
